dali/kernels/imgproc/color_manipulation/brightness_contrast_test.cc

```diff
@@ -15,7 +15,7 @@
 #include <gtest/gtest.h>
 #include <vector>
 #include <tuple>
-#include "dali/kernels/test/tensor_test_utils.h"
+#include "dali/test/tensor_test_utils.h"
```

Comment:
Why `tensor_test_utils.h` goes to tests, while `kernel_test_utils.h` not?

Reply:
Because kernel_test_utils depends much more heavily on what's defined in dali/kernels, while tensor_test_utils - not any more.